src/python/gudhi/persistence_graphical_tools.py

```diff
@@ -183,7 +194,11 @@ def plot_persistence_diagram(
     inf_delta=0.1,
     legend=False,
     colormap=None,
-    axes=None
+    axes=None,
+    aspect_equal=False,
+    fontsize=16,
+    title="Persistence diagram",
+    greyblock=True
```

Comment:
aspect_equal, font_size and grey_block.
I am clearly faulty on this point but we shall homogenize the names.

Reply:
Could we keep `fontsize` this way? This is the variable name used by matplotlib functions by default, and I think we should be consistent with it. Having `fontsize=font_size` is a bit weird to me, and a someone used to matplotlib would naturally go for `fontsize=...` by default.